admin-data-explorer-web/src/main/scala/net/wiringbits/webapp/utils/ui/web/components/EditGuesser.scala

```diff
@@ -24,18 +30,33 @@ object EditGuesser {
         }
       }
 
-    val buttons = tableActionsMaybe
-      .map { tableActions =>
-        tableActions.actions.map { action =>
-          val primaryKey = dom.window.location.hash.split("/").lastOption.getOrElse("")
-          Button(_.onClick := (() => action.onClick(primaryKey)))(action.text)
+    def onClick(action: ButtonAction, ctx: js.Dictionary[js.Any]): Unit = {
+      val primaryKey = dom.window.location.hash.split("/").lastOption.getOrElse("")
+      action.onClick(primaryKey).onComplete {
```

Comment:
Is there a way to show a progress indicator and disable the action button white this is  being processed?